client/app/views/templates/view_steps_password.jade

```diff
@@ -10,11 +10,10 @@ block region
         label(for="password")=t('step password label')
         button(class='icon', action="password-visibility", title=t(visibilityTxt))
             svg: use(xlink:href=visibilityIcon)
-        input(type=inputType, name="password", id="password" placeholder=t("password placeholder"))
+        input(type=inputType, name="password", id="password" placeholder=t("step password input placeholder"))
     progress.pw-weak(value=passwordStrength.percentage max=100)
 
-    if error
-        p(class="errors")= error
+    p(class="errors")
```

Comment:
I like the `if error` condition in the template, but I understand your need here (being able to update easily a persistent error container. I think we should be able to keep the best of the two worlds by using a Subview. Errors need to be handled too in others steps, so we may update this part soon. Let's go with it for now.